agent/util-scripts/tool-meister/pbench-sysinfo-dump

```diff
@@ -11,13 +29,15 @@ if [[ ! -d "${dir}" ]]; then
 	exit 1
 fi
 
+# Which pieces of system information should be collected. The options
+# available are the suffixes to all the collect_* functions.
 sysinfo="${2}"
 
-if [[ -z "${_PBENCH_UNIT_TESTS}" ]]; then
-	mode="${3}"
-else
-	# Force sequential mode for unit tests.
-	mode=""
+# Collection mode, serial or parallel.
+mode="${3}"
+if [[ "${mode}" != "serial" && "${mode}" != "parallel" ]]; then
+	printf -- "Invalid collection mode, \"%s\"\n" "${mode}" >&2
+	exit 1
```

Comment:
So there is no reason to force sequential mode for unit tests? Won't the various collections step on each other? Could it produce a random order in the output file? Or is that sorted? Or is it invoked in sequential mode?

Reply:
Yes, because it is mocked out.  See https://github.com/distributed-system-analysis/pbench/blob/main/agent/util-scripts/test-bin/pbench-sysinfo-dump.